apps/astarte_pairing/lib/astarte_pairing_web/router.ex

```diff
@@ -38,6 +38,8 @@ defmodule Astarte.PairingWeb.Router do
 
     get "/version", VersionController, :show
 
+    post "/ownership", OwnershipController, :create
```

Comment:
1. once again, let's not abbreviate our terms
2. do we plan to add more ownership voucher functions in the future, and/or
3. do we need authorization?


```suggestion
    scope "/ownership-voucher" do
      # pipe_through :ownership_voucher_api ?

      post "/", OwnershipVoucherController, :create
    end
```

(the only multi-word endpoint I could find is in appengine's router, and it uses kebab case: `"/v1/realm/devices-by-alias"`)

Reply:
- do we plan to add more ownership voucher functions in the future -> probably not
- do we need authorization? -> so far, the standard jwt authentication of pairing should be enouth (agent_api plug, i assume?)
-i'll do it in kebab case

Reply:
I suggest having a different pipeline anyway
`agent_api` is clearly made to be `"/agent" specific and is expected to evolve with its needs (though it probably won't)
if the name were `user_api` or `jwt_api` or something else I would have agreed with using it